### DISCUSSION TITLE
nb 菜单怎么打开？

### DISCUSSION THREAD

**Discussion by Conkinya** [nonebot/discussions#457] [问答]
见插件https://github.com/18870/nonebot-plugin-reboot 的要求

**Comment by yanyongyu** [comment]
什么菜单？nb哪有菜单¿

**Reply by Conkinya** [reply]
那个插件里面说的啊

**Reply by Conkinya** [reply]
https://github.com/18870/nonebot-plugin-reboot?tab=readme-ov-file#nonebot-plugin-reboot

**Reply by yanyongyu** [reply]
那是nb-cli的交互式命令行，可以直接运行`nb`命令
